Sources/Controllers/AnimatableTabBarController.swift

```diff
@@ -39,6 +39,19 @@ open class AnimatableTabBarController: UITabBarController, TransitionAnimatable
     }
   }
 
+  // MARK: - UITabBarDelegate
+  open override func tabBar(_ tabBar: UITabBar, didSelect item: UITabBarItem) {
+    guard let animatable = item as? (UITabBarItem & Animatable),
+      let tabBarButton = tabBar.view(for: item) else {
+        return
+    }
+    // Animate the children image view
+    let subviews = tabBarButton.subviews.filter { $0 is UIImageView }
+    if let view = subviews.first {
```

Comment:
I assume it should be only one image inside. That's fine, we can get the first one in one let

```
if let imageView = self.subviews.first(where: { $0 is UIImageView }) {
  animatable.animate(view: imageView)
}
```